.github/workflows/ci.yml

```diff
@@ -45,54 +32,7 @@ jobs:
           - 3.6-stable
```

Comment:
We did not backport the `matrix.json` changes to 3.5-stable at all, and 3.6-stable seems to be missing the PostgrSQL addition in file.

I think it's safe to drop 3.5/3.6 here (and add 3.8/3.9 instead)